core/web/user_controller.go

```diff
@@ -170,7 +170,7 @@ func getETHAccount(ctx *gin.Context, store *store.Store, account accounts.Accoun
 		return presenters.ETHKey{}
 	}
 
-	linkBalance, err := txm.GetLINKBalance(account.Address)
+	linkBalance, err := bulletprooftxmanager.GetLINKBalance(store.Config, store.EthClient, account.Address)
```

Comment:
could we just use txm in the codebase?

Reply:
a very good call - I'll make that change before merging